src/talkaction.cpp

```diff
@@ -130,6 +130,16 @@ TalkActionResult_t TalkActions::playerSaySpell(Player* player, SpeakClasses type
 			}
 		}
 
+		if (it->second.fromLua) {
+			if (it->second.getNeedAccess() && !player->getGroup()->access) {
+				return TALKACTION_CONTINUE;
```

Comment:
shouldn't this be `return TALKACTION_BREAK;` as the player does not meet the required group access?

Reply:
I was thinking that as well but after looking at the other scripts I thought I should keep the same behaviour; 
`player has no access -> return true`
`player has wrong accType -> return false`

Reply:
Yea ok, I can see now what you mean, rest of code looks fine to me now